src/iuse.cpp

```diff
@@ -9488,3 +9488,107 @@ int iuse::weather_tool(player *p, item *it, bool, const tripoint& )
     return 0;
 }
 
+int iuse::capture_monster_act(player *p, item *it, bool, const tripoint& pos)
+{
+    if(it->has_var("contained_name")) {
+        try {
+            tripoint target;
+            if(g->is_empty(pos)) {
+                target = pos;//it's been activated somewhere where there isn't a player or monster, good.=
+            } else {
+                if(it->has_flag("PLACE_RANDOMLY")) {
+                    std::vector<tripoint> valid;
+                    for(int x = p->posx()-1; x <= p->posx() + 1; x++) {
+                        for(int y = p->posy() - 1; y <= p->posy() + 1; y++) {
+                            tripoint dest(x,y,pos.z);
+                            if(g->is_empty(dest)) {
+                                valid.push_back(dest);
+                            }
+                        }
+                    }
+                    if(valid.size() == 0) {
+                        p->add_msg_if_player(_("There is no place to put the %s"),it->get_var("contained_name","").c_str());
+                        return 0;
+                    }
+                    target = random_entry(valid);
+                } else {
+                    const std::string query = string_format(_("Place the %s where?"),it->get_var("contained_name","").c_str());
+                    if(!choose_adjacent(query,target)) {
+                        return 0;
+                    }
+                    if(!g->is_empty(target)) {
+                        p->add_msg_if_player(m_info,_("You cannot place the %s there!"),it->get_var("contained_name","").c_str());
+                    }
+                }
+            }
+            monster new_monster;
+            new_monster.deserialize(it->get_var("contained_json",""));
+            new_monster.spawn(target);
+            g->add_zombie(new_monster);
+            it->erase_var("contained_name");
+            it->erase_var("contained_json");
+            it->erase_var("name");
+            it->erase_var("weight");
+            return 0;
+        } catch(const std::string &e) {
+            debugmsg(_("Error restoring monster: %s"),e.c_str());
+        }
+    } else {
+        tripoint target = pos;
+        const std::string query = string_format(_("Capture what with the %s?"),it->tname(1).c_str());
```

Comment:
The 1 is the default value for that parameter of `item::tname`, you don't need to specific it explicitly.

`it->tname( 1 )` and `it->tname()` are the same and yield the same. Applies to all invocations of that function here.